app/models/external_pull_request.rb

```diff
@@ -26,6 +27,8 @@
 
 class ExternalPullRequest < ApplicationRecord
   belongs_to :external_project
+  alias project external_project
```

Comment:
I think this would work for read-only ops, but what about assignments? Please, disregard this if it isn't something of importance. I'm bringing it up just in case.

Reply:
Yeah, it probably doesn't work for assignments, but I just needed it for read operations anyway (to make `ExternalPullRequest` polymorphic with `PullRequest` for the `project` message).
Do you think it would be best for it to work for assignments as well?

Reply:
If it isn't something in our short-term backlog, then I would leave it as it is and tackle it when it becomes a problem.